relay-profiling/src/outcomes.rs

```diff
@@ -3,13 +3,15 @@ use crate::ProfileError;
 pub fn discard_reason(err: ProfileError) -> &'static str {
     match err {
         ProfileError::CannotSerializePayload => "profiling_failed_serialization",
+        ProfileError::InvalidBase64Value => "profiling_invalid_base64_value",
+        ProfileError::InvalidJson(_) => "profiling_invalid_json",
```

Comment:
We should use the existing discard reasons for this, such as `DiscardReason::InvalidJson` to keep cardinality of the column overall as low as possible. However, we can also follow up about this.